src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -367,6 +373,11 @@ public ErrorDiagnostic CannotResolveFunctionOverload(IList<string> overloadSigna
                 "BCP066",
                 $"Function \"{functionName}\" is not valid at this location. It can only be used in resource declarations.");
 
+            public ErrorDiagnostic ObjectRequiredForMethodAccess(TypeSymbol wrongType) => new ErrorDiagnostic(
+                TextSpan,
+                "BCP067",
+                $"Cannot access functions on type \"{wrongType}\". An \"{LanguageConstants.Object}\" type is required.");

```

Comment:
Should we call it "invoke" or "call" instead of "access"?

Reply:
Yes, that's a much better suggestion :)